src/ISMP.sol

```diff
@@ -42,7 +42,7 @@ abstract contract IsmpHost is IIsmpHost, Context {
     Consensus private _consensus;
 
     // monotonically increasing nonce for outgoing requests
-    uint256 private _nonce;
+    uint64 private _nonce;
```

Comment:
```suggestion
    uint256 private _nonce;
```

prefer we keep this 32 bytes for optimal storage packing

Reply:
Okay. I change it because of the change in the requests.

Please explain how it helps for optimal storage packing?